aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -364,6 +364,11 @@ module aptos_framework::voting {
         proposal_id: u64,
     ) acquires VotingForum {
         let proposal_state = get_proposal_state<ProposalType>(voting_forum_address, proposal_id);
+
```

Comment:
Using assume with ghost state is not an ideal way to specify aborts_if condition. Could you try to write a spec fun version of get_proposal_state instead? If it is difficult to do so, we can visit this aborts_if condition later.